lib/builtin.pm

```diff
@@ -168,6 +168,31 @@ numerical argument.
 Returns the largest integer value less than or equal to the given numerical
 argument.
 
+=head2 indexed
+
+    @kvlist = indexed(@list)
+
+Returns an even-sized list of number/value pairs, formed by taking successive
+values from the list of arguments and prefixing each one with an integer
```

Comment:
prefixing sounds to me like "1value" "2value". I think this whole paragraph is kind of overly confusing. It doesn't  mention array indices which I think are well understood and might make this clearer?


```
Returns an even-sized list of index/value pairs for the given list, in order. For example:

    my @list = qw(first second third);
    my @indexed = indexed @list; # (0, first, 1, second, 2, third);